app/models/resource.rb

```diff
@@ -1,10 +1,11 @@
 class Resource < ActiveRecord::Base
   belongs_to :conference
+  validates :name, :quantity, presence: true
```

Comment:
your task implies that `used` should also be present, so I would validate this as well. 

And I would also like to update our DB accordingly, so that null is **not** allowed for `used` and `quantity`.